moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -790,10 +791,10 @@ bool ServoCalcs::applyJointUpdate(const Eigen::ArrayXd& delta_theta, sensor_msgs
   };
 
   // Transform that applies the lambda to all joints.
-  // joint_state contains the future position q(t + dt)
-  // last_joint_state contains past position q(t - dt)
-  std::transform(joint_state.position.begin(), joint_state.position.end(), last_joint_state_.position.begin(),
-                 joint_state.velocity.begin(), compute_velocity);
+  // next_joint_state contains the future position q(t + dt)
+  // previous_joint_state_ contains past position q(t - dt)
```

Comment:
This comment isn't quite right. We're only iterating forward by one `dt`. I think it should be:

```
  // next_joint_state contains the future position q(t)
  // previous_joint_state_ contains past position q(t - dt)
```